package.json

```diff
@@ -31,20 +31,27 @@
     "src"
   ],
   "scripts": {
+    "build": "tsc",
```

Comment:
You can take a look at the CDR repo for examples of this. I think you update this to follow that:
https://github.com/foxglove/cdr/blob/bcf974da138cf119830ea70787bd42f08b56391a/package.json#L33

just add a `yarn clean` and update the build to be `tsc -b`. Another thing that's good to do is have a separate `tsconfig.build.js` that extends the original `tsconfig` but excludes the tests. This makes it so that the package itself doesn't include the tests when deploying. I've done this with some of the packages in the omgidl repo. See here: https://github.com/foxglove/omgidl/blob/main/packages/omgidl-serialization/package.json